Dockerfile

```diff
@@ -1,36 +1,50 @@
-FROM ubuntu:22.04
+FROM python:3.10-slim-bullseye
```

Comment:
I could not get it to work with `bookworm`. I'm always getting
`oscrypto.errors.LibraryNotFoundError: Error detecting the version of libcrypto` even though `libssl-dev` is installed and openssl is working.